spec/models/reviewable_akismet_post_spec.rb

```diff
@@ -199,8 +200,29 @@ def assert_history_reflects_action(action, admin, post, action_name)
       end
     end
 
-    describe '#perform_confirm_delete' do
-      let(:action) { :confirm_delete }
+    describe '#perform_delete_user' do
+      let(:action) { :delete_user }
+      let(:action_name) { 'confirmed_spam_deleted' }
+      let(:flag_stat_status) { :agreed }
+
+      it_behaves_like 'It logs actions in the staff actions logger'
```

Comment:
This is very minor and an optional follow up, but these "sentences" read oddly.

"It behaves like it logs actions in the staff actions logger" doesn't really explain the behaviour. 

For example:

"It behaves like a staff action logger"
"it behaves like an akismet feedback submission"

Read much more like BDD.